lib/colonel_kurtz/active_record/serializer.rb

```diff
@@ -0,0 +1,39 @@
+module ColonelKurtz
+  module ActiveRecord
+    class Serializer
+      def self.load(value)
+        case value
+        when String
+          value
+        when NilClass
+          ''
+        when Hash, Array
+          value.to_json
```

Comment:
Am I thinking about this wrong, or is there a chance that these are backwards? Intuitively, it makes more sense to me that `load` would be the one calling `JSON.parse`, and `dump` would call `.to_json`.

Reply:
> When loading into Ruby, we actually want a string so it can fit nicely inside of a form value
Is that statement only true when a field is loaded for the admin view? And if that is the case, wouldn't it make more sense that the application code would be more concerned with the JSON->string conversion?

Reply:
Ya, we only want the stringed value when messing with forms.

But, as is, this doesn't really change the types for the user. Right now they get a string, and with this PR they still get a string when calling the method on the instance. 

There's another helper `has_content` that'll add methods to parse it into nice block objects. That still works.

Also, there's no requirement to use the serializer. This PR only makes it available for use.